scripts/agent_step_receipt_relabeling_harness.py

```
@@ -0,0 +1,696 @@
+    bundle["receipt"]["dependency_drop_manifest_commitment"] = commitment_for(
+        bundle["dependency_drop_manifest"],
+        "agent-step-receipt-v1.dependency-drop-manifest",
+    )
+    bundle["receipt"]["evidence_manifest_commitment"] = commitment_for(
+        bundle["evidence_manifest"],
+        "agent-step-receipt-v1.evidence-manifest",
+    )
+    recompute_receipt_commitment(bundle)
+
+
+def _mutate_field(field: str, value: Any) -> Callable[[dict[str, Any]], None]:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)